packages/module/playground/components/ui/carousel/Carousel.vue

```diff
@@ -1,5 +1,4 @@
 <script setup lang="ts">
-import emblaCarouselVue from 'embla-carousel-vue'
```

Comment:
Will Merge this PR, but it's better to ignore changes to that is not related to the PR, maybe next time 😄 

Can you also remove exact same unused imports in `Carousel.vue` **default** and **new-york** styles too?

Thanks

Reply:
sorry, to resolve confict I just merge `dev` into my branch to resolve conflicts locally, how can I do it better next time?

Reply:
Ow yes you right ❤️  my bad

Reply:
ahh I added this import else the registry dependencies can't track it well 😂